ex02/script.js

```diff
@@ -1,14 +1,28 @@
 // Write your Task 1 code inside this function!
 function task1() {
   console.log("Running task 1");
+  // 1. target the div
+  let target = document.querySelector("#text-content");
+  console.log(target);
+  // 2. set inner text to something new
+target.innerText = "Running Task 1"
+  
 }
 
 // Write your task 2 code inside this function!
 function task2() {
   console.log("Running task 2");
+  let target = document.querySelector("#task2")
+  console.log(target)
+  target.classList.toggle("inverted")
+     
+
 }
 
 // Write your task 3 code inside this function!
 function task3() {
   console.log("Running task 3");
+  let target = document.querySelector("#list")
+  console.log(target)
+  target.appendChild(document.createElement("li")).textContent = "Do it again the next day"
```

Comment:
Good job on this line - you don't always have to use variables to access elements on the page. This is called "chaining" where you link several calls together.